README.rst

```diff
@@ -1,11 +1,16 @@
-AWS CloudFormation Resource Provider Development Kit
-====================================================
+AWS CloudFormation CLI
+======================
 
 The CloudFormation CLI (cfn) allows you to author your own resource providers that can be used by CloudFormation.
 
 Usage
 -----
 
+Documentation
+^^^^^^^^^^^^^
+
```

Comment:
is this space intentional? looks a little empty when looking at the rich diff

Reply:
The space *after* the `Documentation` title is following the rest of the doc. The space before it though I can remove. Which do you mean?

Reply:
yes, before